### DISCUSSION TITLE
Jinja Support

### DISCUSSION THREAD

**Discussion by Mar1usW3** [Clooos/Bubble-Card#758] [Feature requests]
It would be great to have jinja support in the styling sections. So it is more consistent with othet cards.

**Comment by nagtegaal** [comment]
Definatly better than the JS option. It would make things more uniform with Home Assistant tempting and opens up a whole new level of possibilities without the need to learn another templating language.

For me Jinja support is a big +1 feature-wise, usability-wise and be more consistent with home assistant core templating. The benefit of this is that you can use the developer tools to test your code, and there is no need to test through out the styling editor and eventually see that your code isn't working at all.

So...

+1 from me

**Comment by Clooos** [comment]
Hi! Jinja templates will finally be available in v3.4.0! You will be able to use these to customize names, icons, sub-buttons and styles 🍻

**Reply by Clooos** [reply]
And done! It was about time! 🍻

https://github.com/Clooos/Bubble-Card/releases/tag/v3.4.0
